pipeline.py

```diff
@@ -0,0 +1,72 @@
+import pmu,common
+from tabulate import tabulate
+import itertools,sys,re
+
+def pipeline_view(log,depths):
+  out_pipe=open(log.replace('csv','pipeline.log'),'w');
+  pipe_list=get_log(log)
+  event_list=[]
+  depth_list=[]
+  lines=[]
+  for i in depths:
+    event_list.append(depths[i][0])
+    depth_list.append(int(depths[i][1]))
+  issued_name=depths['issued'][0] 
+  issued_list=[i for i,item in enumerate(event_list) if re.search(issued_name,item)]
+  issued_index=issued_list[0]
```

Comment:
Since you're not using issued_name or issued_list in any other place in the code, you can combine building the list and defining issued_index in one line.
issued_index = [i for i,item in enumerate(event_list) if re.search(issued_name,item)][0]
(You can even combine the 3 lines in one but it may be too much).

Reply:
fixed